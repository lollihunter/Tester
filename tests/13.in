20 20 20
1 1
20 13
7 6
8 17
19 13
5 4
12 1
5 5
6 19
9 15
10 13
18 9
17 6
11 13
2 1
15 16
14 6
14 8
9 17
20 20
