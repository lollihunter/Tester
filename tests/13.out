10
1 15 6 3 7 13 12 5 16 20 
